components/org.wso2.carbon.status.dashboard.web/src/worker/WorkerSpecific.jsx

```diff
@@ -68,26 +69,58 @@ export default class WorkerSpecific extends React.Component {
 
     componentWillMount() {
         let that = this;
-        AuthenticationAPI.isUserAuthorized('manager', AuthManager.getUser().token)
+        AuthenticationAPI.isUserAuthorized('manager', AuthManager.getUser().SDID)
             .then((response) => {
                 that.setState({
                     hasManagerPermission: response.data
                 });
             }).catch((error) => {
-            let re = /The session with id '((?:\\.|[^'])*)'|"((?:\\.|[^"])*)" is not valid./;
-            let found = error.response.data.match(re);
-            if (found != null) {
+            let message;
+            if(error.response != null){
```

Comment:
proper spacings are missing

Reply:
Fixed in 50c0b3ff27410966326b69c3e721733fe60b591b